projects/rero/ng-core/src/lib/record/editor/toggle-wrapper/toggle-wrappers.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { FieldWrapper } from '@ngx-formly/core';
+import { isEmpty, removeEmptyValues } from '../utils';
+
+@Component({
+  selector: 'ng-core-editor-formly-toggle-wrapper',
+  template: `
+    <div class='toggle-wrapper'>
```

Comment:
I do not find the styles of toggle-wrapper

Reply:
No specific styles are used for `toggle-switch`. 

If `bootstrap toggle` library is loaded for projec, the display will be a switch (like the screenshoot), otherwise a default checkbox will be displayed.